ws-client/src/helpers.rs

```diff
@@ -73,19 +73,19 @@ pub fn process_batch_response(manager: &mut RequestManager, rps: Vec<JsonRpcResp
 ///
 /// Returns `Ok()` if the response was successfully sent to the frontend.
 /// Return `Err(None)` if the subscription was not found.
-/// Returns `Err(Some(msg))` if the subscription was full.
+/// Returns `Err(Some(msg))` if the channel to the frontend was full.
```

Comment:
I don't mind this much but I would call the `subscription stream` as a "frontend" as it's part of the `Subscription` returned to user after a subscription has been established and "not a frontend" in the client....